os/pm/Kconfig

```diff
@@ -42,6 +42,9 @@ config PM_METRICS_DURATION
 	int "History Duration (secs)"
 	default 60
 
+config PM_HISTORY_COUNT
+	int "History Count"
+	default 5
```

Comment:
Could you add `---help---` for explanation about this config?

Reply:
modified